### ISSUE TITLE
Cannot find module '../i18n/undefined.json'

### ISSUE THREAD

**Issue by devakira** [HelloWorld017/atom-discord#71]
**Please use this template only if issue related to Error / Unexpected behavior**  
**Please use suitable tags**

## Step to reproduce
1. Update to atom-discord v2.0.0
2. Restart Atom

## Stacktrace
```
Error: Cannot find module '../i18n/undefined.json'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (~\AppData\Local\atom\app-1.32.1\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:11:154368)
    at Module.require (file:///~/AppData/Local/atom/app-1.32.1/resources/app.asar/static/index.js:40:43)
    at require (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:11:145675)
    at Object.<anonymous> (~\.atom\packages\atom-discord\src\index.js:39:10)
    at Object.<anonymous> (~\.atom\packages\atom-discord\src\index.js:336:3)
    at Module.get_Module._compile (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:11:146385)
    at Object.value [as .js] (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:11:149932)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (file:///~/AppData/Local/atom/app-1.32.1/resources/app.asar/static/index.js:47:45)
    at require (internal/module.js:11:18)
    at customRequire (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:1:622321)
    at Package.requireMainModule (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:11:3044031)
    at measure (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:11:3030742)
    at Package.measure (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:11:3029574)
    at Package.load (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:11:3030315)
    at PackageManager.loadAvailablePackage (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:11:378592)
    at config.transact (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:11:377015)
    at Config.transact (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:11:329551)
    at PackageManager.loadPackages (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:11:376953)
    at t.loadState.then (~\AppData\Local\atom\app-1.32.1\resources\app\static\<embedded>:1:642340)
    at <anonymous>
```

## Information
**Platform**: Microsoft Windows 10 Home 10.0.17134 Build 17134
**Atom**: 1.32.1
**Package Version**: v2.0.0

### Non-Core Packages
```
Community Packages (15) ~\.atom\packages
├── atom-discord@2.0.0
├── atom-html-preview@0.2.5
├── atom-live-server@2.3.0
├── atom-material-syntax@1.0.8
├── atom-material-ui@2.1.3
├── atom-ternjs@0.18.3
├── file-icons@2.1.26
├── minimap@4.29.9
├── minimap-cursorline@0.2.0
├── minimap-find-and-replace@4.5.2
├── minimap-pigments@0.2.2
├── pigments@0.40.2
├── sync-settings@0.8.6
├── teletype@0.13.3
└── wakatime@7.0.7
```

**Comment by KingFaris10** [comment]
I get the same issue with far less packages too.

**Comment by HelloWorld017** [comment]
Please check bugs are existing on v2.0.1.

**Comment by devakira** [comment]
Can confirm this is fixed in 2.0.1
